### PR TITLE
Improved countries modification

### PR DESCRIPTION
**Pull request by roadster31**
This PR improves countries management, and remove the obsolete "Area" (e.g. Shipping Zone) field.

The form field renderer does no longer escape the 'help' field, thus allowing to put links and other code in help text.

### CODE REVIEW

File: local/modules/Front/Config/front.xml [hunk 1]
```diff
@@ -222,7 +222,7 @@
     <!-- end contact management -->
 
     <!-- newsletter management -->
-    <route id="newsletter.process" path="/newsletter" methods="post">
+    <route id="newsletter.process" path="/newsletter">
```
Comment (lunika) [inline local/modules/Front/Config/front.xml]: why did you remove the methods ?

Comment (roadster31) [inline local/modules/Front/Config/front.xml]: I don't remember.
